### ISSUE TITLE
Adapt for use with smlpkg

### ISSUE THREAD

**Issue by athas** [kfl/wpp#1]
It would be nice if this was available via [smlpkg](https://github.com/diku-dk/smlpkg).

**Comment by kfl** [comment]
A PR would be welcome 😉 

@melsman also asked for this, so I guess there is an urgent demand.

I'm definitely willing, but I would prefer if the package could stay here. Is that possible?

**Comment by athas** [comment]
Of course.  `smlpkg` just requires that the repository directory structure is of a specific form ([example](https://github.com/diku-dk/sml-md5)).

**Comment by kfl** [comment]
Ugh, do I really need to have a Russian-dool directory structure where the current git-hosting is part of the naming of directories 🤮  Would it be more intuitive with the SHA256 sum?

I'll look at it.

**Comment by melsman** [comment]
.. we can also just do a fork...

lør. 20. mar. 2021 kl. 11.07 skrev Ken Friis Larsen <
***@***.***>:
